docs/tutorials/t2ramsey_tutorial.ipynb

```diff
@@ -0,0 +1,416 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# set the computation units to seconds\n",
+    "unit = 'us' #seconds\n",
+    "qubit = 0\n",
+    "# set the desired delays\n",
+    "delays = list(range(1, 150, 2))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "     ┌───┐┌──────────────┐┌──────┐ ░ ┌───┐ ░ ┌─┐\n",
+      "q_0: ┤ H ├┤ DELAY(1[us]) ├┤ P(0) ├─░─┤ H ├─░─┤M├\n",
+      "     └───┘└──────────────┘└──────┘ ░ └───┘ ░ └╥┘\n",
+      "c: 1/═════════════════════════════════════════╩═\n",
+      "                                              0 \n"
+     ]
+    }
+   ],
+   "source": [
+    "# Create a T2Ramsey experiment. Print the first circuit as an example\n",
+    "exp1 = T2Ramsey(qubit, delays, unit=unit)\n",
+    "print(exp1.circuits()[0])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We run the experiment on a simple, simulated backend, created specifically for this experiment."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from qiskit_experiments.test.t2ramsey_backend import T2RamseyBackend\n",
+    "# MockJob is a wrapper for the backend, to give it the form of a job\n",
+    "from qiskit_experiments.test.mock_job import MockJob\n",
+    "import qiskit_experiments.matplotlib\n",
+    "from qiskit_experiments.matplotlib import pyplot, requires_matplotlib\n",
+    "from qiskit_experiments.matplotlib import HAS_MATPLOTLIB\n",
+    "\n",
+    "conversion_factor = 1E-6\n",
+    "# The behavior of the backend is determined by the following parameters\n",
+    "backend = T2RamseyBackend(\n",
+    "                    p0={\"a_guess\":[0.5], \"t2ramsey\":[80.0], \"f_guess\":[0.02], \"phi_guess\":[0.0],\n",
+    "                        \"b_guess\": [0.5]},\n",
+    "                    initial_prob_plus=[0.0],\n",
+    "                    readout0to1=[0.02],\n",
+    "                    readout1to0=[0.02],\n",
+    "                    conversion_factor=conversion_factor,\n",
+    "                )\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The resulting graph will have the form:\n",
+    "$ f(t) = a^{-t/T_2*} \\cdot cos(2 \\pi f t + \\phi) + b $\n",
+    "where *t* is the delay, $T_2*$ is the decay factor, and *f* is the detuning frequency.\n",
+    "`conversion_factor` is a scaling factor that depends on the STI units used. It is 1E-6 here, because the unit is microseconds."
```

Comment:
Explain (or just expand the acronym) what is STI

Reply:
ok, done.